README.md

```diff
@@ -53,7 +56,8 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(n). This is because it should check starting from both the start and end of the sentence and meet in the middle, 
+which is O(n/2), which is equal to O(n).
 
```

Comment:
Yup (although in reality it starts checking from the middle: it builds the stack, checking at each step if the rest of the string matches the stack when popped)